waspc/data/Generator/templates/react-app/src/auth/buttons/Google.js

```diff
@@ -0,0 +1,9 @@
+import config from '../../config.js'
+
+export default function Google(props) {
+  return (
+    <a href={`${config.apiUrl}/auth/external/google/login`}>
+      <img alt="Google" height={props?.height || 40} src="/images/btn_google_signin_dark_normal_web@2x.png" />
```

Comment:
Do we want to give them more flexibility, like a different image, or use text instead? Or just wait for now until we get that request?